packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -31,6 +31,9 @@ export interface ITelemetryLoggerPropertyBags{
  * Creates sub-logger that appends properties to all events
  */
 export abstract class TelemetryLogger implements ITelemetryLogger {
+    // supportsTags remains optional, but now all child classes of TelemetryLogger must handle it properly if set
+    abstract supportsTags?: true | undefined;
```

Comment:
I think the comment is a little confusing, may not be necessary at all.  The confusing part is "if set" -- it's not how they set it themselves, but how it's set on any other loggers they may interact with, but that's not relevant to the value they assign to it.

Also, remember that the `?` and `| undefined` are redundant.

Reply:
oops